qiskit_ibm/runtime/ibm_runtime_service.py

```diff
@@ -107,17 +107,23 @@ def __init__(self, provider: 'ibm_provider.IBMProvider') -> None:
         self._ws_url = provider.credentials.runtime_url.replace('https', 'wss')
         self._programs = {}  # type: Dict
 
-    def pprint_programs(self, refresh: bool = False) -> None:
+    def pprint_programs(self, refresh: bool = False, detailed: bool = False) -> None:
         """Pretty print information about available runtime programs.
 
         Args:
             refresh: If ``True``, re-query the server for the programs. Otherwise
                 return the cached value.
+            detailed: If ``True`` print all details about available runtime programs.
         """
         programs = self.programs(refresh)
         for prog in programs:
-            print("="*50)
-            print(str(prog))
+            if detailed:
+                print("="*50)
```

Comment:
`print("="*50)` can be moved above the if/else block since it's used in both if and else block?